retweets.go

```diff
@@ -51,3 +52,89 @@ func retweetsLookup(ctx context.Context, c *client, id string, opt ...*RetweetsL
 
 	return &retweetsLookup, nil
 }
+
+type Body struct {
+	TweetID string `json:"tweet_id"`
+}
```

Comment:
please move it to tweet.go

```suggestion
type TweetBody struct {
	TweetID string `json:"tweet_id"`
}
```